### PR TITLE
doc: add debug listener example in operations guide

### PR DESCRIPTION
**Pull request by szuecs**
Signed-off-by: Sandor Szücs <sandor.szuecs@zalando.de>

### CODE REVIEW

File: docs/operation/operation.md [hunk 1]
```diff
@@ -761,3 +761,87 @@ basis with the rfcPath() filter. See
 If the second interpretation gets considered the right way, and the
 other one a bug, then the default value for this flag may become to
 be on.
+
+## Debugging Requests
+
+Skipper provides [filters](../reference/filters.md), that can change
```
Comment (aryszka) [inline docs/operation/operation.md]: I would mention that sometimes the most important information that it shows which request is matched.

Comment (szuecs) [reply docs/operation/operation.md]: I changed the text to explain what it shows

**Comment by aryszka** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Pull request merged**